thoth/prescriptions_refresh/handlers/scorecards.py

```diff
@@ -862,14 +862,17 @@ def scorecards(prescriptions: "Prescriptions") -> None:
     _LOGGER.info("Mapping available prescriptions to scorecards")
     client = bigquery.Client()
 
+    job_labels = {"query_target": "openssf-scorecards"}
+    job_config = bigquery.QueryJobConfig(labels=job_labels)
+
     for project_name, organization, repository in iter_gh_info(prescriptions):
-        query_job = client.query(
-            f'SELECT * FROM openssf.scorecardcron.scorecard WHERE Repo="github.com/{organization}/{repository} "'
-            "AND Date=(SELECT MAX(Date) FROM openssf.scorecardcron.scorecard "
-            f'WHERE Repo="github.com/{organization}/{repository}") '
-            f"AND Date > DATE_ADD(CURRENT_DATE(), interval"
-            f" -{_THOTH_PRESCRIPTIONS_REFRESH_SCORECARD_FRESHNESS_WEEKS} week)"
-        )
+        query = f"""
```

Comment:
```suggestion
        query = f'
```

Maybe we should avoid double quotes both inside and outside the query. Have you tested it yet?

Reply:
Yes, [triple] double quotes work here, and multi-line strings can contain other quotes.
I could change to single quotes, but we are using doubles everywhere else, so I think it's better to keep that as is.

Reply:
Moreover, I just verified that `black` would switch back `'''` to `"""`, so let's leave it with double quotes